app/src/main/java/io/apicurio/registry/storage/impl/sql/CommonSqlStatements.java

```diff
@@ -254,7 +254,7 @@ public String selectArtifactVersionContent() {
      */
     @Override
     public String selectArtifactContentIds() {
-        return "SELECT v.contentId FROM versions v WHERE v.tenantId = ? AND v.groupId = ? AND v.artifactId = ?";
+        return "SELECT v.contentId FROM versions v WHERE v.tenantId = ? AND v.groupId = ? AND v.artifactId = ? ORDER BY v.versionId";
```

Comment:
I think a better field to order by would be `globalId`. I think we are not using `versionId` anywhere else other than in the IBM API and I'm not confident it works properly.

Reply:
Looking at the code again it seems should work.